scripts/badger/bridge_out_arbitrum_fees/remove_liq_swapr.py

```diff
@@ -0,0 +1,59 @@
+from rich.console import Console
+from brownie import interface, chain, Contract, accounts
+from helpers.addresses import registry
+
+CONSOLE = Console()
+
+# dxs tokens to liquidated
+dxs_tokens = ["dxsWbtcWeth", "dxsBadgerWeth"]
```

Comment:
i also see dxsSwprWeth (`0xA66b20912cBEa522278f3056B4aE60D0d3EE271b`) and dxsIbbtcWeth (`0x6a060a569e04a41794d6b1308865a13F27D27E53`) balances on the msig.

Reply:
good catch!